infrastructure/main.tf

```diff
@@ -46,6 +46,7 @@ module "case-management-web" {
     CCD_GW_LOGOUT_URL = "${local.ccd_gateway_url}/logout"
     CCD_API_URL = "${local.ccd_gateway_url}/aggregated"
     CCD_DATA_URL = "${local.ccd_gateway_url}/data"
+    CCD_DRAFT_URL = "${local.ccd_gateway_url}/data"
```

Comment:
Do we need to add DRAFT_URL? I thought we are always going to go through data store DATA_URL?